compiler/rustc_ast_pretty/src/pprust/state.rs

```diff
@@ -2659,8 +2659,12 @@ impl<'a> State<'a> {
                     s.word_space(":");
                     s.print_type(ty);
                     s.print_type_bounds(":", &param.bounds);
+                    // FIXME(const_generic_defaults)
                     if let Some(ref _default) = default {
                         // FIXME(const_generics_defaults): print the `default` value here
+                        s.s.space();
+                        s.word_space("=");
+                        // s.print_anon_const(&default);
```

Comment:
As I mentioned before, I had done https://github.com/lqd/rust/commit/e779f69a1c45212ea199d36ade2059b0f9fb24b8 and the associated test https://github.com/lqd/rust/commit/2498ccac67c54cd9355c42ec5f1160c62c1f3d28 in my branch if you wanted some inspiration (the test still contains a function with defaulted const generics, which has since been made invalid IIUC)

Reply:
has it been merged and I just accidentally deleted it while rebasing?

Reply:
no it's more in-progress work I had after extracting the parsing from this PR, and which tries to fix other issues in diagnostics (some shared with defaulted types), add tests, etc that I mention could be of interest in the first paragraph of https://github.com/rust-lang/rust/pull/75384#issuecomment-797080201

Reply:
this saved me some of trouble, thanks lqd!